docs(README): Updates, make more precise (#58)
Superformula/test-web-stack: README.md

While this README assumes you implement both backend and frontend, only frontend context is required if you apply for Front-end position. To better showcase your skills and experience we encourage you to complete both "UI Context" and "Backend Context" parts depending on your time availability. In case you decide not to implement backend according to spec, feel free to implement mock api to accompany your frontend work.

> - **Please update readme with instructions on how to run a project, ideally we should be able to run it with one command (please also include all needed environment variables, or send them by email if possible).**

Reviewer: ```suggestion
- **Remember to update README with instructions on how to run a project. Ideally, we should be able to run it with one command. Please also include all required environment variables, or send them by email.**
```

> - **After opening a pull request, please send us an email with link to that pull request.**

Reviewer: ```suggestion
- **After opening a pull request, please email us a link to it.**
```

> - If you have anything that you would like to improve in your app, feel free to add “Improvements” section in readme and write it all there.

Reviewer: ```suggestion
- If you have anything that you would like to improve in your app in case of more time, feel free to add “Improvements” section in README and outline it all there.
```

## UI context

> Use HTML, CSS, and JavaScript (choose one of popular framework) to implement the following mock-up. You are only required to complete the desktop views, unless otherwise instructed. Application should connect to created GraphQL API. If you have a limited amount of time, it’s better to create one fully tested feature (covering accessibility, unit tests, e2e tests, e.g. edit user functionality) instead of whole app finished partially.

Reviewer: ```suggestion
Use HTML, CSS, and JavaScript (choose one of popular framework) to implement the following mock-up. You are only required to complete the desktop views, unless otherwise instructed. Application should connect to created GraphQL API. If you have a limited amount of time, it is preferred you create one fully covered feature (including accessibility, unit tests, e2e tests, storybook stories, e.g. edit user functionality) instead of the partially implemented whole app.
```

### Bonus (in order)

> 1. Providing an online demo is welcomed (e.g. using vercel)

Reviewer: ```suggestion
1. Providing an online demo is welcomed (e.g. using Vercel or Netlify)
```

1. Delete user functionality

## What We Care About

> **We expect the application to look the same as designs (sizes, spacings, fonts, icons, etc), with attention to details.**

Reviewer: I don't think we need this sentence here since we mention this down below. I'd rather say this instead as the first paragraph, WDYT?

```
We expect you treat this assignment as if it were a production app with everything implied you do in a production grade project.
```

Use any libraries that you would normally use if this were a real production App. Please note: we're interested in your code & the way you solve the problem, not how well you can use a particular library or feature.

_We're interested in your method and how you approach the problem just as much as we're interested in the end result._

Here's what you should strive for:

> - **Application to look the same as designs (sizes, spacings, fonts, icons, etc), with attention to details.**

Reviewer 2: NIT: Application _should_ look the same as designs ...

Author: I'll live it as it is as:
 `"...you should strive for:" "Application to look the same as designs"` sounds more decent than
 `"...you should strive for:" "Application should look the same as designs" `

- Solid testing approach
- Extensible code

> - Application to look same as designs

Reviewer: ```suggestion
- Application to look exactly the same as designs (sizes, spacings, fonts, icons, etc.)
```
